build.gradle

```diff
@@ -5,14 +5,14 @@ buildscript {
         jcenter()
     }
 
-    ext.kotlin_version = '1.3.72'
+    ext.kotlin_version = '1.4.10'
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.0'
+        classpath 'com.android.tools.build:gradle:4.0.1'
         classpath 'com.google.gms:google-services:4.3.3'
-        classpath 'org.ow2.asm:asm:7.3.1' // https://github.com/jacoco/jacoco/issues/639#issuecomment-355424756
-        classpath 'org.jacoco:org.jacoco.core:0.8.5'
-        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.1.0'
+        classpath 'org.ow2.asm:asm:9.0' // https://github.com/jacoco/jacoco/issues/639#issuecomment-355424756
```

Comment:
Did you investigate if we still need this comment? I'm wondering if we don't as Jacoco is mostly disabled at the moment.

Reply:
Right, it seems like we don't need it anymore. I removed it.